luarules/gadgets/cmd_build_bugger_off.lua

```diff
@@ -0,0 +1,186 @@
+local gadget = gadget ---@type gadget
+
+function gadget:GetInfo()
+    return {
+        name    = "Builder buggeroff",
+        desc    = "Enables busy builders and moving units to buggeroff",
+        author  = "Flameink",
+        date    = "March 7, 2025",
+        version = "1.0",
+        license = "GNU GPL, v2 or later",
+        layer   = 0,
+        enabled = true   --  loaded by default?
+    }
+end
+
+local debugLog = true
+
+function print(Message)
+   if debugLog then
+        Spring.Echo(Message)
+    end
+end
+
+local shouldNotBuggeroff = {}
+local cachedUnitDefs = {}
+for unitDefID, unitDef in pairs(UnitDefs) do
+    if unitDef.isImmobile then
+        shouldNotBuggeroff[unitDefID] = true
+    end
+    cachedUnitDefs[unitDefID] = { radius = unitDef.radius, isBuilder = unitDef.isBuilder}
+end
+
+if gadgetHandler:IsSyncedCode() then
```

Comment:
If there is no unsynced part then everything (perhaps except `GetInfo`) should be under the synced check. In particular the unsynced part shouldn't build the `cachedUnitDefs` table.